draft-schinazi-connect-udp-listen.md

```diff
@@ -66,12 +66,12 @@ allows proxying UDP payloads {{!UDP=RFC0768}} to a fixed host and port. Combined
 with the HTTP CONNECT method (see {{Section 9.3.6 of !HTTP=RFC9110}}), it allows
 proxying the majority of a Web Browser's HTTP traffic. However WebRTC {{WebRTC}}
 relies on ICE {{?ICE=RFC8445}} to provide connectivity between two Web browsers,
-and that in turn relies on the ability to send and receive UDP packets to
-multiple hosts. While it would be possible in theory to accomplish this by using
+and which relies on the ability to send and receive UDP packets to
```

Comment:
`that` is more correct than `which` here, since the subsequent clause is essential to the meaning of the sentence
```suggestion
and that relies on the ability to send and receive UDP packets to
```

Reply:
I would actually go with removing the `and`:

```suggestion
which relies on the ability to send and receive UDP packets to
```